chain/src/main/java/org/ethereum/beacon/chain/BeaconChainListener.java

```diff
@@ -0,0 +1,8 @@
+package org.ethereum.beacon.chain;
+
+import java.util.concurrent.CompletableFuture;
+
+/** A stub for beacon chain listener. */
+public interface BeaconChainListener {
```

Comment:
Can be replaced by rx `Publisher` later (streams have a notion of `Mono` - 0|1 element stream)